packages/ns-don/Makefile

```diff
@@ -29,6 +29,17 @@ define Package/ns-don/description
 	Connect to a remote Windmill installation
 endef
 
+define Package/ns-dedalo/firewall_config
```

Comment:
Not sure, is `ns-dedalo` correct?

Reply:
:scream: I do not how it can works, but it does.
I will fix it